Gaijinizer patch/Map465.txt

```diff
@@ -3,493 +3,634 @@
 > END STRING
 
 > BEGIN STRING
 全ての物事が解決した後なら
 
 \\C[2]『アバドーンがもたらした被害を全て修復する』権限\\C[0]が
 彼女には与えられてるって事
-> CONTEXT: Map465/events/3/pages/0/197/Dialogue/sirousagi_fc1 < UNTRANSLATED
-
+> CONTEXT: Map465/events/3/pages/0/197/Dialogue/sirousagi_fc1
+She's been assigned \\C[2]the authority to fix all
+harm brought about by Abaddon\\C[0] after everything
+has been dealt with.
 > END STRING
 
 > BEGIN STRING
 アバドーンがもたらした被害……
 
 \\C[2]アバドーンに喰われて破壊された世界を直せる\\C[0]って事か…！
-> CONTEXT: Map465/events/3/pages/0/202/Dialogue/kazuya < UNTRANSLATED
+> CONTEXT: Map465/events/3/pages/0/202/Dialogue/kazuya
+Harm brought about by Abaddon, huh...
 
+\\C[2]So you're saying it's possible to repair worlds
+that've been consumed and destroyed by Abaddon?!#Ron
 > END STRING
 
 > BEGIN STRING
 大当たり！！
 
 まぁ『後処理』の権限だから、ちゃんと事象を解決しないと
 施行できない権限だけどね……
-> CONTEXT: Map465/events/3/pages/0/206/Dialogue/sirousagi_fc1 < UNTRANSLATED
-
+> CONTEXT: Map465/events/3/pages/0/206/Dialogue/sirousagi_fc1
+Bingo!!
+But since it only authorizes her to deal with the
+aftermath specifically, it won't be of much help
+until everything is properly resolved.#Ron
 > END STRING
 
 > BEGIN STRING
 …………ということは、もし解決できなければ……
-> CONTEXT: Map465/events/3/pages/0/211/Dialogue/alice_fc5 < UNTRANSLATED
-
+> CONTEXT: Map465/events/3/pages/0/211/Dialogue/alice_fc5
+So, if we can't resolve everything...#Ron
 > END STRING
 
 > BEGIN STRING
 被害はそのまま、アバドーンは引き続き世界を躍り食い…
 
 当然じゃん？
-> CONTEXT: Map465/events/3/pages/0/213/Dialogue/sirousagi_fc1 < UNTRANSLATED
-
+> CONTEXT: Map465/events/3/pages/0/213/Dialogue/sirousagi_fc1
+Obviously the damage will remain, and Abaddon
+will move on to devour the next world.#Ron
 > END STRING
 
 > BEGIN STRING
 …………私に選択肢はないみたいね。
 
 自分の世界がなくなってしまった以上
 帰る場所もないみたいだし……
-> CONTEXT: Map465/events/3/pages/0/218/Dialogue/enemy38 < UNTRANSLATED
+> CONTEXT: Map465/events/3/pages/0/218/Dialogue/enemy38
+Doesn't seem like I have much of a choice.
 
+Now that my own world has been destroyed, it
+seems I don't even have anywhere to go back to.#Ron
 > END STRING
 
 > BEGIN STRING
 協力させてもらうわ、\\N[1]さん。
 
 そのアバドーンとやらをぶっ潰すために…！
-> CONTEXT: Map465/events/3/pages/0/223/Dialogue/enemy38 < UNTRANSLATED
+> CONTEXT: Map465/events/3/pages/0/223/Dialogue/enemy38
+Let me help, \\N[1]-san.
 
+We'll crush this Abaddon!
 > END STRING
 
 > BEGIN STRING
 \\N[1]でいいよ……
 
 いや、中学生だからさん付けが妥当なんだろうけど…
-> CONTEXT: Map465/events/3/pages/0/227/Dialogue/kazuya < UNTRANSLATED
+> CONTEXT: Map465/events/3/pages/0/227/Dialogue/kazuya
+You can just call me \\N[1].
 
+Then again, since you're a middle-schooler it
+might be more appropriate for you to use "san".#Ron
 > END STRING
 
 > BEGIN STRING
 ルカといいアリスといい、なんで見た目と雰囲気の年齢が
 
 かけ離れてる面子ばかりなんだろう……この一行……
-> CONTEXT: Map465/events/3/pages/0/231/Dialogue/kazuya < UNTRANSLATED
-
+> CONTEXT: Map465/events/3/pages/0/231/Dialogue/kazuya
+Why is it that Luka, Alice, and everyone else
+in this party feel so much older than their
+appearance would suggest?#CheckLater
 > END STRING
```

Comment:
Note

Reply:
Luka is usually drawn as a shota in MGQ but ultimately sits at around 18. Alice is explicitly 18, but White Rabbit placed a seal on her that forced her to split off as a smaller, weaker, younger-looking body. So Luka and Alice are adults that look like children, while Mizuki is a child that looks like an adult. ...I'm not entirely sure whether Kazuya's comment is based on perceiving Mizuki as older as she is, or Mizuki acting older than she is.

>Luka, Alice, and now you.
>Why is this party filled with people who look
>(far younger/totally different) than they act?

I think this could work.